runtime/bindings/python/src/pyopenvino/graph/rt_map.cpp

```diff
@@ -20,29 +20,19 @@
 
 namespace py = pybind11;
 
-using PyRTMap = std::map<std::string, std::shared_ptr<ov::Variant>>;
+using PyRTMap = std::map<std::string, ov::Any>;
 
 PYBIND11_MAKE_OPAQUE(PyRTMap);
 
-template <typename T>
-void _set_with_variant(PyRTMap& m, const std::string& k, const T v) {
-    auto new_v = std::make_shared<ov::VariantWrapper<T>>(ov::VariantWrapper<T>(v));
-    auto it = m.find(k);
-    if (it != m.end())
-        it->second = new_v;
-    else
-        m.emplace(k, new_v);
-}
-
 void regclass_graph_PyRTMap(py::module m) {
-    auto py_map = py::bind_map<PyRTMap>(m, "PyRTMap");
+    auto py_map = py::bind_map<PyRTMap>(m, "PyRTMap", py::module_local());
     py_map.doc() = "ngraph.impl.PyRTMap makes bindings for std::map<std::string, "
                    "std::shared_ptr<ov::Variant>>, which can later be used as ov::Node::RTMap";
 
     py_map.def("__setitem__", [](PyRTMap& m, const std::string& k, const std::string v) {
-        _set_with_variant(m, k, v);
+        m[k] = v;
     });
     py_map.def("__setitem__", [](PyRTMap& m, const std::string& k, const int64_t v) {
-        _set_with_variant(m, k, v);
+        m[k] = v;
```

Comment:
If we are going to set items only as `int64_t` and `std::string` is it against the idea of `ov::Any`? You can create some type of cast from `py::object` since this will represent all (at least most of all) python's object.

Reply:
Frankly speaking, I am not familiar nither with python nor with `pybind`. The python part of this PR just tries to preserve the workability of the current PyAPI using `ov :: Any`

Reply:
If in OV 2.0 the RTMap is supposed accept value of any type (because underneath it uses ov::Any) then this constraint here makes no sense. You can probably remove both definitions of `__setitem__` to be able to feed any value to RTMap from python scripts.

Reply:
I am going to preserve current behavior but move python API changes due to `ov::Any` into separate activity